ingestclient/core/engine.py

```diff
@@ -231,10 +237,12 @@ def monitor(self, workers):
                         log_str += " - Elapsed time {:.2f} minutes".format((time.time() - start_time) / 60)
                         always_log_info(log_str)
                     else:
-                        always_log_info("Waiting to ensure all upload tasks have been processed. Just a few minutes longer...")
+                        always_log_info(
```

Comment:
Are you using a IDE that is reformatting lines longer than a given length? Just an odd way of doing this (and the one above and below)

Why not create a log_msg string with the value and when just pass it to the logger method?

Reply:
updated.